samples/Localization.StarterWeb/wwwroot/lib/bootstrap/dist/js/bootstrap.min.js

```diff
@@ -0,0 +1,7 @@
+/*!
```

Comment:
We probably want to leave most of this js stuff in grunt/bower rather than checking it in right?